src/middleware/notFound.middleware.ts

```diff
@@ -0,0 +1,8 @@
+import type { NextFunction, Request, Response } from 'express';
+
+const routeNotFound = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
- rename `routeNotFound` -> `routeNotFoundHandler` as its handle unknown routes // just for naming convention